octane-eclipse-plugin/src/com/hpe/octane/ideplugins/eclipse/ui/entitydetail/EntityFieldsComposite.java

```diff
@@ -148,7 +148,10 @@ private void drawEntityFields(Set<String> shownFields, EntityModel entityModel)
 
         // make a map of the field names and labels
         Collection<FieldMetadata> allFields = metadataService.getVisibleFields(Entity.getEntityType(entityModel));
-        allFields.stream().filter(f -> !f.getName().equals(EntityFieldsConstants.FIELD_DESCRIPTION));
+        allFields.stream()
+                .filter(f -> !f.getName().equals(EntityFieldsConstants.FIELD_DESCRIPTION)
+                        && !f.getName().equals(EntityFieldsConstants.FIELD_PHASE)
+                        && !f.getName().equals(EntityFieldsConstants.FIELD_NAME));
```

Comment:
Stream filter doesn't modify the original stream, allFields would need to be re-assigned for that